src/modules/certificate/certificate.contoller.ts

```
@@ -0,0 +1,91 @@
+import { Controller, Post, Body, Res, UseGuards, Req } from '@nestjs/common';
```

ESLint

[error] 'UseGuards' is defined but never used.

(@typescript-eslint/no-unused-vars)